packages/ckeditor5-engine/src/conversion/downcasthelpers.js

```diff
@@ -1349,6 +1366,21 @@ function downcastElementToElement( config ) {
 
 	return dispatcher => {
 		dispatcher.on( 'insert:' + config.model, insertElement( config.view ), { priority: config.converterPriority || 'normal' } );
+
+		if ( config.triggerBy ) {
+			if ( Array.isArray( config.triggerBy.attributes ) ) {
```

Comment:
Why Array.isArray and not simply `!config.triggerBy.attributes` ?

Reply:
Because I wanted to be sure it is an array. But we do not test for this after all so an early Exception on weird config might be better.